js/task.js

```diff
@@ -1,3 +1,8 @@
-var text = document.getElementById('text');
-var button = document.getElementById('button');
-console.log(text, button);
+
+function myFunction() {
+  var text = document.getElementById('text').value;
+  var para = document.createElement("P");
+  var t = document.createTextNode(text);
+  para.appendChild(t);
+  document.getElementById("tag").appendChild(para);
```

Comment:
Each time on submut btn you looking for this tag. Move this logic outside from function

Reply:
if i move var, it doesn't work